src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Aes.PlatformNotSupported.cs

```diff
@@ -6,19 +6,26 @@
 namespace System.Runtime.Intrinsics.Arm
 {
     /// <summary>
-    /// This class provides access to the ARM AES hardware instructions via intrinsics
+    /// Provides access to the ARM AES hardware instructions via intrinsics
     /// </summary>
     [CLSCompliant(false)]
     public abstract class Aes : ArmBase
     {
         internal Aes() { }
 
+        /// <summary>Gets a value that indicates whether the APIs in this class are supported.</summary>
```

Comment:
I think this should instead say something along the lines of "Always returns false, indicating that the APIs in this class are not supported."

Reply:
Same suggestion for all the similar *.NotSupported.cs IsSupported properties that return false.

Reply:
We have 1 set of docs that exist, not separate ones for `*.cs` for `*.PlatformNotSupported.cs`

The `*.PlatformNotSupported.cs` are purely implementation details for corelib to be more optimal